tools/mapping/Makefile

```diff
@@ -9,6 +9,7 @@ $(STEP_1_PREREQS):
 	$(call check_prereqs,$(STEP_1_DEPS))
 
 # TODO: Find a way to generate and link config.csh on each run.
+# Chris, there were environmental variables hard coded in mk_SCRIPgridplio_enh.csh. Let's ditch the script, and run the executable here directly.
 $(STEP_1_TARGETS): $(STEP_1_DEPS) 
```

Comment:
# add the following to step 1
cat >! input.scrip_grid << EOF
$nx $ny  # nx, ny are from config.h
${ocngridname}
${popgridfile}
${kmtgridfile}
${ocngridfile}
1
EOF
${scripdir}/grids/myconvertPOPT < input.scrip_grid